### DISCUSSION TITLE
Pensar em integrar o Storybook

### DISCUSSION THREAD

**Discussion by andrewarosario** [techexpertspro/portal-ascepa#10] [Ideas]
Discussão para integração do Storybook no projeto, com objetivo de documentar os componentes.
- [Documentação oficial](https://storybook.js.org/)
- [Setup do Nx para projetos Angular](https://nx.dev/recipes/storybook/overview-angular)
- [https://medium.com/@ortegavan/introdução-ao-storybook-um-exemplo-prático-com-angular-394a1bf61b06](https://medium.com/@ortegavan/introdu%C3%A7%C3%A3o-ao-storybook-um-exemplo-pr%C3%A1tico-com-angular-394a1bf61b06)
- [Repositório Open Source](https://github.com/juntossomosmais/atomium/)

**Comment by ortegavan** [comment]
Pra quem quiser ter um primeiro contato com a ferramenta, escrevi este post a um tempo atrás: https://medium.com/@ortegavan/introdução-ao-storybook-um-exemplo-prático-com-angular-394a1bf61b06

**Comment by andrewarosario** [comment]
Convertido na issue #143
